src/main/java/Event.java

```diff
@@ -0,0 +1,45 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime time;
+
+    /**
+     * Creates an Event object.
+     * 
+     * @param description The description of the event.
+     * @param time The time of the event.
+     */
+    public Event(String description, LocalDateTime time) {
+        super(description);
+        this.time = time;
+    }
+
+    /**
+     * Returns the message to be saved in the hard disk.
+     * 
+     * @return The string representation of the task in the local file.
+     */
+    @Override
+    public String writeMessage() {
+        String done = "";
+        if (this.isDone) {
+            done = "✓";
+        } else {
+            done = "✗";
+        }
+        return String.format("E | %s | %s", done, this.description, this.time);
+    }
```

Comment:
Perhaps you can refactor this method to the Task class to avoid repetition of code in child classes?

Reply:
Agreed. You can write a getter method to get the type of the Task and just move the writeMessage (would be better if you change the name to "getMessage") to Task class. It's more OOP that way 👍